src/components/admin/ScraperDashboard.tsx

```
@@ -0,0 +1,230 @@
+    }
+  };
+
+  return (
+    <div className="space-y-8">
+      {/* Control Panel */}
+      <div className="p-6 bg-white dark:bg-gray-800 rounded-xl shadow-md space-y-4">
+        <h2 className="text-xl font-semibold">Start Scraper</h2>
+        
+        <div className="grid grid-cols-1 md:grid-cols-2 gap-4">
+          <div>
+            <label className="block text-sm font-medium mb-1">Mode</label>
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)